docs/source/common/weights_loading.rst

```diff
@@ -4,32 +4,34 @@
     from pytorch_lightning.trainer.trainer import Trainer
     from pytorch_lightning.core.lightning import LightningModule
 
-.. _weights_loading:
+.. _checkpointing:
 
-##########################
-Saving and loading weights
-##########################
+##############################
+Saving and Loading Checkpoints
+##############################
 
-Lightning automates saving and loading checkpoints. Checkpoints capture the exact value of all parameters used by a model.
+Lightning provides functions to save and load checkpoints.
 
 Checkpointing your training allows you to resume a training process in case it was interrupted, fine-tune a model or use a pre-trained model for inference without having to retrain the model.
 
 
 *****************
-Checkpoint saving
+Checkpoint Saving
```

Comment:
Checkpoint contents? This section is also useful/relevant for loading, not just saving